webapp/channels/src/components/markdown_image/index.ts

```diff
@@ -2,15 +2,21 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
+import {ActionCreatorsMapObject, bindActionCreators, Dispatch} from 'redux';
+import {Action, GenericAction} from 'mattermost-redux/types/actions';
 
 import {openModal} from 'actions/views/modals';
+import {ModalData} from 'types/actions';
 
 import MarkdownImage from './markdown_image';
 
-function mapDispatchToProps(dispatch) {
+type Actions = {
```

Comment:
you can import this part of Props defined in 'markdown_image' component. Instead of redefining again here.
see https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/components/add_user_to_channel_modal/index.ts